src/App.js

```diff
@@ -1,12 +1,188 @@
+    const [appels, setAppels] = React.useState(0);
+    const [kiwis, setKiwis] = React.useState(0);
+
+    const [voornaam, setVoornaam] = React.useState('');
+    const [achternaam, setAchternaam] = React.useState('');
+    const [leeftijd, setLeeftijd] = React.useState(0);
+    const [postcode, setPostcode] = React.useState('');
+    const [bezorging, togglebezorging] = React.useState('week');
+    const [BezorgTijdslot, toggleBezorgTiidslot] = useState('daytime');
+    const [opmerking, setOpmerking] = React.useState('Plaats hier je opmerkingen');
+    const [termenAkkoord, toggletermenAkkoord] = React.useState(false);
+
+// hier zet ik mijn functies neer.
+    function resetFruit() {
+        setAardbeien(0);
+        setBananen(0);
+        setAppels(0);
+        setKiwis(0);
+    }
+
+    function handleClick() {
+        console.log(`
+    Voornaam: ${voornaam}, 
+    Achternaam: ${achternaam}, 
+    Leeftijd: ${leeftijd}, 
+    Postcode: ${postcode}, 
+    Bezorgfrequentie: ${bezorging},
+    Opmerkingen: ${opmerking},
+    Algemene voorwaarden: ${termenAkkoord}
+    `);
+        console.log(`Fruitmand bestelling - aardbeiden: ${aardbeien}, bananen: ${bananen}, appels: ${appels}, kiwi's: ${kiwis}`);
+
+
+//Hier wordt de app gestart
+        return (
+            <>
+                <h1>Fruitmand bezorgservice</h1>
+
+
+                <article className="counter">
+                    <h2> Bananen</h2>
+                    <Counter
+                        fruitTeller={bananen}
+                        setFruitteller={setBananen}>
+                    </Counter>
+                </article>
+
+                <article className="counter">
+                    <h2> Aardbeien</h2>
+                    <Counter
+                        fruitTeller={aardbeien}
+                        setFruitteller={setAardbeien}>
+
+                    </Counter>
+                </article>
+
+                <article className="counter">
+                    <h2> Appels</h2>
+                    <Counter
+                        fruitTeller={appels}
+                        setFruitteller={setAppels}>
+
+                    </Counter>
+                </article>
+
+                <article className="counter">
+                    <h2> Kiwis</h2>
+                    <Counter
+                        fruitTeller={kiwis}
+                        setFruitteller={setKiwis}>
+
+                    </Counter>
+                </article>
+
+                <Button type="button" clickHandler={resetFruit}>Reset</Button>
+
+
+                {/*// Hier komt mijn formulier*/}
+
+                <form className="formulier" onSubmit={handleSubmit(onSubmit)}>
+
+                    <label htmlFor="voornaam">Voornaam</label>
+                    <input type="text" id="Naam"
+                           placeholder="Naam"
+                           value={voornaam}
+                           onChange={() => setVoornaam(e.target.value)}/>
```

Comment:
`e` is hier undefined. Hier moet volgens mij het volgende staan:
```suggestion
                           onChange={e => setVoornaam(e.target.value)}/>
```